src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -52,14 +52,25 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHA, MHASelect,
                                  ::testing::Values(std::map<std::string, std::string>{})),
                          MHA::getTestCaseName);
 
-const std::vector<std::vector<ov::PartialShape>> inputShapesWOTranspose = {
-        {{1, 12, 197, 64}, {1, 12, 64, 197}, {1, 12, 197, 64}},
-        {{1, 12, 12, 64}, {1, 12, 64, 48}, {1, 12, 48, 64}}
-};
+
+static std::vector<std::vector<ov::PartialShape>> inputShapesWOTranspose(bool supports_3d = false) {
```

Comment:
Looks like the better solution is to have two separate vectors (like `inputShapesWOTranspose` and `inputShapesWOTranspose4D`) than unites them on each iteration.

Reply:
Fixed in [PR](https://github.com/openvinotoolkit/openvino/pull/15647) bounds